pkg/cfn/builder/templates/beta-resources.yaml

```diff
@@ -0,0 +1,101 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: CloudFormation template to create an EKS cluster using a custom Lambda function with a unique name.
+
+Parameters:
+  EksEndpointUrl:
+    Type: String
+    Description: The endpoint URL for the EKS service
+
+Resources:
+  ControlPlane:
+    Type: Custom::EksCluster
+    Properties:
+      ServiceToken: !GetAtt CustomEKSFunction.Arn
+      RoleArn: !GetAtt ServiceRole.Arn
+
+  CustomEKSFunction:
+    Type: AWS::Lambda::Function
+    Properties:
+      FunctionName: !Sub '${AWS::StackName}-CustomEKSFunction'
+      Handler: index.handler
+      Runtime: python3.9
```

Comment:
Any reason to pick python 3.9? It's going to be deprecated later this year.

Reply:
Happy to fix this!